### DISCUSSION TITLE
Rookie question about websites

### DISCUSSION THREAD

**Discussion by TheFogy** [tectalichq/public-openai-client-php#2] [Q&A] [answered]
I have zero knowledge of programming. I'm making a website for test/fun(it's an online shop with canvases) and I'm wondering if there are possibilities to test this on the website or if there would be too many changes in the code to do so. Just a loose question, I don't want to make anyone angry if I'm disrespectful, to ask this I'm sorry.

The website is on WordPress

**Comment by thejamescollins** [comment, marked as answer]
Hi there @TheFogy, welcome!

This OpenAI API Client is written for PHP applications, so it could be used in a WordPress-built website because WordPress runs on PHP.

I presume you are using WooCommerce for the online store portion of your website?

You would likely need to build a WordPress plugin that would implement querying the OpenAI API. Here is some information on developing a WordPress plugin:

https://developer.wordpress.org/plugins/

That step would require writing PHP code, and bundling the `tectalic/openai` client within your custom WordPress plugin.

I hope that helps!

James
Tectalic

**Reply by TheFogy** [reply]
Dear @om4james,

Thank you very much, yes it runs on WooCommerce, I guess I have to go back to my Technical School times and learn some PHP again :D 
Thank You so much! 

Fog
